docs/details/commands/pmm-admin.md

```diff
@@ -344,18 +362,43 @@ For example, if you want all collectors except `topmetrics`, specify:
 
 ###### Limit dbStats, collStats and indexStats
 
- All the collectors are enabled by default. However, collStats and indexStats are enabled if there are less than 200 collections across all the databases, excluding the admin, config, and local system databases.
+By default PMM decides what limit to set for the number of collections to monitor for the collStats and indexStats collectors. 
 
-For collStats and indexStats you can:
+If you decide to set another limit you could do that with `--max-collections-limit` parameter.
 
-- Set the value of the parameter `-max-collections-limit` to 0, which indicates that collStats and indexStats can handle unlimited collections.
- 
-Enable collStats and indexStats for some databases or collections also.
+Set the value of the parameter `--max-collections-limit` to:
+
+- 0: which indicates that collStats and indexStats can handle unlimited collections.
+- n, which indicates that collStats and indexStats can handle <=n collections. If the limit is crossed - exporter stops collecting monitoring data for the collStats and indexStats collectors.
+- -1 (default) don't need to be explicitly set, indicates that PMM decides how many collections it would monitor, currenlty <=200, but subject to change.
```

Comment:
-1 (default) doesn't need to be explicitly set, indicates that PMM decides how many collections it would monitor, currently <=200, but subject to change.